idp_node/mapapp/openid_retriever.py

```diff
@@ -0,0 +1,92 @@
+import argparse
+import ConfigParser
+import logging
+import os
+import sys
+
+import OpenSSL
+import psycopg2
+
+def get_openid(args, config_parser, esgf_home):
+
+    pass_file = os.path.join(esgf_home, "config", ".esg_pg_pass")
+    with open(pass_file, "rb") as filep:
+        password = filep.read().strip()
+
+    section = "installer.properties"
+
+    db = config_parser.get(section, "db.database")
+    host = config_parser.get(section, "db.host")
+    port = config_parser.get(section, "db.port")
+    user = config_parser.get(section, "db.user")
+    esgf_host = config_parser.get(section, "esgf.host")
+    gateway = "https://{}/esgf-idp/openid/%".format(esgf_host)
+
+    cs = "dbname=%s user=%s password=%s host=%s port=%s" % (db, user, password, host, port)
+    cs_no_pass = "dbname=%s user=%s host=%s port=%s" % (db, user, host, port)
+    logging.info(cs_no_pass)
```

Comment:
This logging message could have additional context.  Something like "fetching OpenID for {some_user}"